### PR TITLE
Fix for "Sliders in examples don't react properly to clicks"

### PR DESCRIPTION
**Pull request by karandeepSJ**
This PR fixes issue #1454 
- A callback was added to call `translate_green_cube` on `"LeftButtonPressEvent"`. 
- Priority given to ` self.line_click_callback` in `ui.py` to ensure that the position of disk gets updated before moving the cube.

### CODE REVIEW

File: dipy/viz/ui.py [hunk 1]
```diff
@@ -1851,7 +1851,7 @@ def handle_events(self, actor):
 
         """
         self.add_callback(self.slider_line, "LeftButtonPressEvent",
-                          self.line_click_callback)
+                          self.line_click_callback,1)
```
Comment (ranveeraggarwal) [inline dipy/viz/ui.py]: Add a space after the `,` in accordance with PEP8.

File: doc/examples/viz_ui.py [hunk 2]
```diff
@@ -143,14 +143,17 @@ def translate_green_cube(i_ren, obj, slider):
...
 
-
 line_slider = ui.LineSlider2D(initial_value=-2,
                               min_value=-5, max_value=5)
 
 line_slider.add_callback(line_slider.slider_disk,
                          "MouseMoveEvent",
                          translate_green_cube)
 
+line_slider.add_callback(line_slider.slider_line,
```
Comment (ranveeraggarwal) [inline doc/examples/viz_ui.py]: Can you do the same for the disk slider also? Let's merge both these into this PR.

**Comment by dmreagan** [comment]
Please also update *viz_advanced.py*. Once that is done, I think this is ready to merge.

**Comment by skoudoro** [comment]
is it ready to go @dmreagan?

**Pull request merged**